### PR TITLE
cc 13

### PR DESCRIPTION
**Pull request by ZenAtomsk**
- [x] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/multiBracketValidation/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+change string to array, iterate through array.
+push open brackets into a stack, when closing bracket found, compare to top of stack.
+
+O(n) time & space
+
+## Solution
+
+- [Multi Bracket Validation](multi-bracket-validation.js) 
+
+![Whiteboard](assets/codechallenge13whiteboard.png)
```
Comment (adrianhuebner) [inline javascript/code-challenges/multiBracketValidation/README.md:17]: Totally okay to time box yourself and take your time with it! When you have some time to come back to this, get code and some tests written, and I will regrade for you!
